t/plugin/hmac-auth.t

```diff
@@ -1246,16 +1246,16 @@ location /t {
 GET /t
 --- response_body
 uri: /uri
-host: 127.0.0.1
+content-length: 52
 content-type: application/x-www-form-urlencoded
-x-real-ip: 127.0.0.1
+host: 127.0.0.1
 user-agent: lua-resty-http/0.14 (Lua) ngx_lua/10017
```

Comment:
a little confuse, why `user-agent` is in this place ?

Reply:
When requesting the URL of `/uri`, all request headers will be output. So it is normal to have the request header `user-agent`.

Reply:
@Firstsawyou 
Can we omit the `user-agent`? Since this field will change across different versions.

Reply:
> When requesting the URL of `/uri`, all request headers will be output. So it is normal to have the request header `user-agent`.

I mean it is not sorted in order.

Reply:
> > When requesting the URL of `/uri`, all request headers will be output. So it is normal to have the request header `user-agent`.
> 
> I mean it is not sorted in order.

It has been sorted, `user-agent` is after `host`.  ^ _ ^